### DISCUSSION TITLE
`coveo update` does NOT work when installed with `npm`

### DISCUSSION THREAD

**Discussion by akaustav** [coveo/cli#285] [Q&A] [answered]
Hi,
A few days ago, I had installed `@coveo/cli` on my work laptop (running Windows 10) using the `npm install -g @coveo/cli` command (I had installed `node-gyp` before that). During that installation, v1.3.0 of Coveo CLI was installed on my machine. I noticed that a [`v1.5.0`](https://github.com/coveo/cli/releases/tag/v1.5.0) was released yesterday - **June 22, 2021**. So, I tried to follow instructions documented in the [Usage section](https://github.com/coveo/cli/tree/v1.5.0#usage) to upgrade Coveo CLI:

> Afterwards, anytime you run `coveo update`, your CLI installation will also update to the latest version.

I ran the `coveo update` command in a command prompt and received a `not updatable` message. Is there a different recommended way to upgrade Coveo CLI for folks who installed the `npm` way?

```text
C:\>coveo version
@coveo/cli/1.3.0 win32-x64 node-v14.17.0

C:\>coveo update
@coveo/cli: Updating CLI... not updatable

C:\>coveo version
@coveo/cli/1.3.0 win32-x64 node-v14.17.0
```

Ameet

**Comment by louis-bompart** [comment, marked as answer]
Hi @akaustav!

 I reproduced your issue, this is definitely a bug. I created an issue here #290.
 
 In the meantime, as a workaround, I'd advise you to run `npm un -g @coveo/cli` followed by `npm i -g @coveo/cli` (or just `npm i -g @coveo/cli@latest`.
 It should help you get through.
 
 Thanks for your report!

**Reply by akaustav** [reply]
Thank you, @louis-bompart for confirming that this is indeed a widespread issue and not just an issue with me. And thanks for opening issue #290. I did the full un-install and re-install of @coveo/cli like you described. That works. however, I had a separate issue with that process which I posted in question #286.
